ale/drivers/co_drivers.py

```diff
@@ -329,6 +329,29 @@ def frame_chain(self):
         return self._frame_chain
 
 class CassiniIssIsisLabelIsisSpiceDriver(Framer, IsisLabel, IsisSpice, NoDistortion, Driver):
+
+    id_lookup = {
+        "ISSNA" : "CASSINI_ISS_NAC",
+        "ISSWA" : "CASSINI_ISS_WAC"
+    }
+
+    name_lookup = {
+        "ISSNA" : "Imaging Science Subsystem Narrow Angle Camera",
+        "ISSWA" : "Imaging Science Subsystem Wide Angle Camera"
+    }
```

Comment:
Can we make these look ups part of the module rather than member variables in the class? Similar to the viking drivers. This would also need to happen for the CassiniIssPds3LabelNaifSpice